apps/cli/src/commands/build.ts

```diff
@@ -19,6 +19,7 @@ type ImageInfo = {
     env: string[];
     ramSize: string;
     sdkVersion: string;
+    sdkType: string;
```

Comment:
I suggest making this variable consistent with the label.

-  `sdkName`
- `CARTESI_LABEL_SDK_NAME`
- `io.cartesi.rollups.sdk_name`